src/selectors/visibleBreakpoints.js

```diff
@@ -46,8 +45,8 @@ function isVisible(breakpoint, selectedSource) {
   */
 export const getVisibleBreakpoints = createSelector(
```

Comment:
Do you know how to add a flow type to getVisibleBreakpoints?  I think the issue with the "disable other breakpoints" has to do with a Breakpoint being compared to a FormattedBreakpoint from this selector that gets a false negative.

Reply:
hmm, i forgot. what do we we do with other memozed selectors?